packages/instant-meilisearch/__tests__/filter.test.ts

```diff
@@ -174,4 +147,34 @@ describe('Instant Meilisearch Browser test', () => {
     const hits = response.results[0].hits
     expect(hits[0].title).toEqual('Ariel')
   })
+
+  test('numeric filter', async () => {
```

Comment:
I'm curious: why do you have to change the numeric filter handling?

Reply:
That's because we want to escape the attribute, as it might contain special characters. The old regular expression (`filter.replace(/(.*):(.*)/i, '"$1"="$2"')`) didn't match [numeric comparisons](https://www.algolia.com/doc/api-reference/api-parameters/numericFilters/#numeric-comparisons), so nothing was escaped.